jarbas/urls.py

```diff
@@ -27,7 +28,8 @@
     path('api/chamber_of_deputies/',
          include(
              'jarbas.chamber_of_deputies.urls',
-             namespace='chamber_of_deputies'))
+             namespace='chamber_of_deputies')),
+    path('healthcheck/', lambda _: HttpResponse(), name='healthcheck'),
```

Comment:
Maybe instead of only testing that the HTTP server answer requests, we could hit the DB (and possibly Celery)? I'm thinking on a simple query, like `SELECT 1;`, just to ensure the DB is responsive.

Reply:
Great idea for the implementation ; ) Gonna code that soon, but feel free to send a PR to this branch.